extensions/podman/src/podman-install.ts

```diff
@@ -367,6 +368,10 @@ export class PodmanInstall {
         }
       } else if (answer === 'Ignore') {
         this.podmanInfo.ignoreVersionUpdate = updateInfo.bundledVersion;
+      } else if (answer === 'Open release note') {
+        await extensionApi.env.openExternal(
+          extensionApi.Uri.parse(`https://github.com/containers/podman/releases/tag/v${updateInfo.bundledVersion}`),
```

Comment:
just wondering if the link should not be part of the podman5.json file

here it's quite an hidden information

Reply:
I moved it to the `podman5.json` file the href